### PR TITLE
Add utility users

### PR DESCRIPTION
**Pull request by Olaniyan99**

### CODE REVIEW

File: src/users/userInteractor.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { UserStore } from './dataStore';
+
+function getUsersStore() {
+    return UserStore.getInstance();
+}
+
+export async function getUsers() {
+    return await getUsersStore().getUsers();
```
Comment (pzalep1) [inline src/users/userInteractor.ts]: @nvisal1 Should this be tested?

**Comment by pzalep1** [comment]
It looks like the build is failing. I would check with the circle ci config to make sure that the changes you made to the docker file are also in circle

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+# ENV PATH /opt/node_modules/.bin:$PATH
+
+# WORKDIR /user/app
+# COPY . /user/app
+
+# # Build source and clean up
+# RUN npm run build
+# RUN npm install
+
+# CMD ["npm", "start"]
```
Comment (nvisal1) [inline Dockerfile]: Why is this commented out?

File: src/drivers/MongoDriver.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ export class MongoDriver {
       } else {
         reportError(error);
         return Promise.reject(
-          `Problem connecting to database at ${dbURI}. \n ${error}`
+          `Problem connecting to database at ${dbURI}. \n ${error}`,
```
Comment (nvisal1) [inline src/drivers/MongoDriver.ts]: We should throw errors instead of rejecting strings.

File: src/drivers/express/ExpressRouteAuthDriver.ts [hunk 4]
```diff
@@ -25,5 +26,10 @@ export class ExpressRouteAuthDriver {
             res.sendStatus(code);
         }
       });
+
+    router.get('/utility-users', async(req, res) => {
+        const users = await userInterractor.getUsers();
```
Comment (nvisal1) [inline src/drivers/express/ExpressRouteAuthDriver.ts]: I don't see changes indicating that this route is now requires authentication and authorization

**Pull request merged**